app/models/validators/tribunal_decision_sub_category_validator.rb

```diff
@@ -0,0 +1,31 @@
+class TribunalDecisionSubCategoryValidator < ActiveModel::EachValidator
+
+  def validate_each(validator, attribute, sub_category)
+
+    unless prefixed_by_parent_category? sub_category, validator
+      message = "change to be a sub-category of '#{category_label(validator)}' OR change Category"
+      validator.errors.add attribute, message
+    end
+  end
+
+  private
+
+  def prefixed_by_parent_category? sub_category, validator
```

Comment:
Method args should be wrapped in parenthesis as per [the styleguide](https://github.com/alphagov/styleguides/blob/master/ruby.md#syntax).